docs/docs/testing-components-with-graphql.md

```diff
@@ -249,10 +249,45 @@ export const Header = props => (
 export default Header
 ```
 
+If you prefer to utilize `useStaticQuery` instead, you can rewrite
+the Header component as:
+
+```jsx:title=src/components/header.js
+import React from "react"
+import { StaticQuery, graphql } from "gatsby"
+
+export const PureHeader = ({ data }) => (
+  <header>
+    <h1>{data.site.siteMetadata.title}</h1>
+  </header>
+)
+
+export const Header = props => (
+  const data = useStaticQuery(graphql`
+    query HeaderQuery {
```

Comment:
`HeaderQuery` is new in this change - we need this here? 
if yes: so then it should be mentioned in the text 
if no: then we should leave it off to avoid confusion that this is mandatory

Reply:
Thanks! I removed it.